packages/documentation/docs/controls/_toast_code.md

```diff
@@ -15,22 +15,22 @@ import ApiToastConfigReact from './\_toast/react/toast-config.md';
 
 <Playground
 name="toast" height="18rem"
-examplesByName></Playground>
+></Playground>
 
 ## Custom toast message
 
 <Playground
 name="toast-custom" height="18rem"
 hideInitalCodePreview
-examplesByName></Playground>
+></Playground>
```

Comment:
General: Do we have a format for the attributes? Sometimes we stack them in one line, sometimes every attributes has a single line (+ident)?